PWGJE/Tasks/jetTutorialSkeleton.cxx

```diff
@@ -124,8 +138,13 @@ struct JetTutorialSkeletonTask {
     registry.fill(HIST("h_jet_ntracks"), jet.tracks().size());
     double angularity = 0.0;
     for (auto& jetConstituent : jet.tracks_as<aod::Tracks>()) {
-      angularity += jetConstituent.pt() * TMath::Sqrt(TMath::Power(jetConstituent.phi() - jet.phi(), 2.0) + TMath::Power(jetConstituent.eta() - jet.eta(), 2.0));
+      
+      double dPhi = jetConstituent.phi() - jet.phi();
+      registry.fill(HIST("h_dPhi"), AngularDistance(dPhi));
+      
+      angularity += jetConstituent.pt() * TMath::Sqrt(TMath::Power(dPhi, 2.0) + TMath::Power(jetConstituent.eta() - jet.eta(), 2.0));
```

Comment:
`dPhi` is not constrained when used in the distance calculation. It looks like a bug.
Also use [`RecoDecay::sqrtSumOfSquares`](https://github.com/AliceO2Group/O2Physics/blob/master/Common/Core/RecoDecay.h#L93) instead.